src/components/notes-form/notesForm.css

```diff
@@ -0,0 +1,38 @@
+.form--container {
+  z-index: 30;
```

Comment:
`z-index` won't work on `display:static` which is the default.
Kindly mention a `position` property

Reply:
Thanks @znjs  for the review. Here I have used the position property in the component library's modal class. Here only I am increasing the Z- index value.